### DISCUSSION TITLE
Can't leave Capslock

### DISCUSSION THREAD

**Discussion by thecatwasnot** [manna-harbour/miryoku#29] [Q&A] [answered]
I'm using the latest miryoku branch on an ErgoDox Infinity with Qwerty layout and VI style navigation, when capslock mode is entered it seems the only way to get out of it is to reboot the computer.  I'd be happy to disable capslock entirely as I never use it.

**Comment by manna-harbour** [comment, marked as answer]
If you press caps lock it turns on, but if you press it again it doesn't turn off? There's nothing special about the caps lock in this keymap, it's just the standard QMK keycode. Can you try this keyboard on a different computer, and try a different keyboard (or keymap) on this computer, and test the caps lock behaviour?

**Comment by manna-harbour** [comment]
Let me know if this is still an issue.
